pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -46,7 +46,7 @@ let
   # The wrapper scripts use 'cat' and 'grep', so we may need coreutils.
   coreutils_bin = if nativeTools then "" else getBin coreutils;
 
-  default_cxx_stdlib_compile=optionalString (targetPlatform.isLinux && !(cc.isGNU or false))
+  default_cxx_stdlib_compile=optionalString (!nativeTools && targetPlatform.isLinux && !(cc.isGNU or false))
```

Comment:
`gcc` should usually not receive these arguments. Do you mean clang?

Reply:
Do I mean clang where? In this case it's just about when `cc` is null (so we don't really know what the compiler is).